### PR TITLE
fix sporadic errors with bs-modal in tests

### PR DESCRIPTION
**Pull request by larsklevan**
If I create and close a modal in a test I get two sporadic errors. Investigated this because our app's CI has been failing sporadically with the `scrollTop` error.

Here is the test I used to reproduce the issues:

```javascript
let i;
for (i = 0; i < 5000; i++) {
  test(`bs-modal remove markup ${i}`, function(assert) {
    assert.expect(0);
    this.set('showModal', true);
    this.render(hbs`{{#bs-modal open=showModal onHide=(action (mut showModal) false) as |modal|}}
        {{#modal.body}}Hello world!{{/modal.body}}
      {{/bs-modal}}`);
    this.set('showModal', false);
  });
}
```

error w/ "scrollTop"
screen shot 2017-11-11 at 1 40 20 pm

error w/ "set on destroyed object"
screen shot 2017-11-11 at 3 05 05 pm

### CODE REVIEW

File: addon/components/base/bs-modal.js [hunk 1]
```diff
@@ -418,7 +418,9 @@ export default Component.extend(TransitionSupport, {
       this.set('inDom', true);
       schedule('afterRender', () => {
         let modalEl = this.get('modalElement');
-        modalEl.scrollTop = 0;
```
Comment (simonihmig) [inline addon/components/base/bs-modal.js]: Why did you remove this line? Shouldn't it just be moved after the guard you added?

**Comment by simonihmig** [comment]
Thanks for working on this! Seems to fix #477.

File: addon/components/base/bs-modal.js (hunk 1 above)
Comment (larsklevan) [reply addon/components/base/bs-modal.js]: Oops - that was an accident. Updated.

**Comment by simonihmig** [comment]
Thanks again! 👍

**Comment by bryanhickerson** [comment]
@simonihmig Can you cut a new release with this fix? I'm hitting this error pretty regularly.

**Comment by simonihmig** [comment]
Sure, will do at the latest on Monday!
